executorlib/base/executor.py

```
@@ -161,7 +165,5 @@ def __del__(self):
         """
         Clean-up the resources associated with the Executor.
         """
-        try:
+        with contextlib.suppress(AttributeError, RuntimeError):
```

Ruff (0.8.2)

Undefined name `contextlib`

(F821)